src/features/create-quiz/create-quiz-question-list/components/question-item.tsx

```diff
@@ -3,14 +3,18 @@ import { Card } from "primereact/card";
 import { Button } from "primereact/button";
 
 import { observer } from "mobx-react";
+import { QuizStore } from "../../store/CreateQuizStore";
+
 interface QuestionItemProps {
   questionName: string;
   handleClick: any;
-  numberOfAnswers: number;
+  handleSelect: any;
+  numberOfAnswers: number | undefined;
```

Comment:
When a type could be undefined this means that it's an optional proprerty, thus can be simplified to just: `numberOfAnswers?: number;`